### PR TITLE
Feature/fix userflow and hover

### PR DESCRIPTION
**Pull request by kubes2020**
# Description
Inside "mission-control" the hover effect is more drastic, the user-flow is specifically Read/Draw/Write in that order due to components being greyed out and onClick disabled
Fixes #
User-flow and hover effect

- What work was done? opacity changes, increase in rem upon hover, grey out tasks that are not available
- Why was this work done? To create better user-flow and UX
- What feature / user story is it for? N/A
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/MissionControl/RenderMissionControl.js [hunk 1]
```diff
@@ -49,23 +49,23 @@ const RenderMissionControl = props => {
...
     if (!hasDrawn && hasRead) {
       push('/child/drawing-sub');
     }
   };
+  const handleWrite = e => {
+    e.stopPropagation();
+    if (!hasWritten && hasRead && hasDrawn) {
+      push('/child/writing-sub');
+    }
+  };
```
Comment (tompsherman) [inline src/components/pages/MissionControl/RenderMissionControl.js:68]: def huge user flow error... will help make game play more intuitive and create less working errors

**Review by tompsherman — APPROVED** [review]

File: src/components/pages/MissionControl/RenderMissionControl.js (hunk 1 above)
Comment (AprilAshby) [reply src/components/pages/MissionControl/RenderMissionControl.js:68]: I agree. This flow is much better for gameplay and will prevent users misunderstanding the drawing portion.

**Review by AprilAshby — APPROVED** [review]
Thanks for making the user experience more intuitive!

**Comment by LizDrumm** [comment]
Looks great! Will solve a lot of confusion.

**Pull request merged**